android/app/build.gradle

```diff
@@ -1,6 +1,7 @@
 apply plugin: 'com.android.application'
 apply plugin: 'de.undercouch.download'
 apply plugin: 'com.google.firebase.crashlytics'
+apply plugin: 'kotlin-android'
```

Comment:
What is this needed for?

Reply:
integrated some kotlin code, but then decided against it, so not really needed. i'll remove